packages/api/src/beacon/routes/debug.ts

```diff
@@ -115,6 +150,24 @@ export function getDefinitions(_config: ChainForkConfig): RouteDefinitions<Endpo
         onlySupport: WireFormat.json,
       },
     },
+    getDebugForkChoice: {
+      url: "/eth/v1/debug/fork_choice",
+      method: "GET",
+      req: EmptyRequestCodec,
+      resp: {
+        data: ForkChoiceResponseType,
+        meta: EmptyMetaCodec,
+        onlySupport: WireFormat.json,
+        transform: {
```

Comment:
as per spec it's not wrapped in data container... it's too late to change it now though :cry: